tests/integration/two-active-chatters.spec.hc.js

```diff
@@ -208,17 +208,17 @@ orchestrator.registerScenario('Two Active Chatters', async scenario => {
       newMessage.channel = channelInFocus.entry
       newMessage.entry.content = 'Hello from Alice, the native holochain user on the shared network. :)'
 
-      // alice (web) sends a message
+      // alice (web) sends a message      
       await page.focus('textarea')
       await page.keyboard.type(newMessageContent(), { delay: 100 })
       // press 'Enter' to submit
       page.keyboard.press(String.fromCharCode(13))
 
       const checkNewMessageState = () => callRegistry['chat.create_message']
-      await waitForState(checkNewMessageState, 'done')
+      await waitForState(checkNewMessageState, 'done', 'chat.create_message', () => callRegistry)
```

Comment:
What does `() => callRegistry` do? It looks like it would do nothing since we're just calling the passed function and not using it's return value within `waitForState`

Reply:
It's just used for improved logging in the polling loop.